From 948c47122cca0897e824dee064b3640c40b83ca6 Mon Sep 17 00:00:00 2001
From: Stephen Gallagher <sgallagh@redhat.com>
Date: Mon, 13 Apr 2009 12:55:07 -0400
Subject: [PATCH] Build fixes for RPM packaging of SSSD

We were missing several BuildRequires for the autotools. Also, we
were linking against two external libraries in the common code
that we do not actually use.
---
 common/collection/Makefile.am |    2 +-
 common/ini/Makefile.am        |    2 +-
 sssd.spec                     |    4 ++++
 3 files changed, 6 insertions(+), 2 deletions(-)

diff --git a/common/collection/Makefile.am b/common/collection/Makefile.am
index a47f8f9..c47327d 100644
--- a/common/collection/Makefile.am
+++ b/common/collection/Makefile.am
@@ -16,4 +16,4 @@ include_HEADERS = collection.h collection_tools.h
 # Build unit test
 noinst_PROGRAMS = collection_ut
 collection_ut_SOURCES = collection_ut.c
-collection_ut_LDADD = libcollection.la -lm -lz
+collection_ut_LDADD = libcollection.la
diff --git a/common/ini/Makefile.am b/common/ini/Makefile.am
index 962172f..7b5324d 100644
--- a/common/ini/Makefile.am
+++ b/common/ini/Makefile.am
@@ -16,4 +16,4 @@ include_HEADERS = ini_config.h
 # Build unit test
 noinst_PROGRAMS = ini_config_ut
 ini_config_ut_SOURCES = ini_config_ut.c
-ini_config_ut_LDADD = libini_config.la -lm -lz ../collection/libcollection.la
\ No newline at end of file
+ini_config_ut_LDADD = libini_config.la ../collection/libcollection.la
diff --git a/sssd.spec b/sssd.spec
index 88da56f..31e18f7 100644
--- a/sssd.spec
+++ b/sssd.spec
@@ -24,6 +24,9 @@ Requires(postun): /sbin/service
 ### Build Dependencies ###
 
 BuildRequires: autoconf
+BuildRequires: automake
+BuildRequires: libtool
+BuildRequires: m4
 BuildRequires: popt-devel
 BuildRequires: libtalloc-devel
 BuildRequires: libtevent-devel
@@ -35,6 +38,7 @@ BuildRequires: openldap-devel
 BuildRequires: pam-devel
 BuildRequires: nss-devel
 BuildRequires: nspr-devel
+BuildRequires: pcre-devel
 
 %description
 Provides a set of daemons to manage access to remote directories and
-- 
1.6.0.6

